File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int positive_input(void);

int main(void)
{

int n= positive_input();
for (int i=1; i<n+1 ;i++)
{
for (int j=0;j<n-i;j++)
{
printf(" ");
}
for (int k=0;k<i;k++)
{
printf("#");
}
printf("  ");
for (int z=0;z<i;z++)
{
printf("#");
}

printf("\n");
}

}


int positive_input(void)
{
int n;
do
{
n= get_int("Height: ");
}
while (n<1 || n>8);

return n;
}

File: pset6/mario/more/mario.py
from cs50 import get_int

while True:
    s= get_int("Height: ")
    if s >0 and s<9:
        break

for i in range(s):
    for j in range(s-i-1):
        print(" ", end="")
    
    for z in range(i+1):
        print("#", end="")
    
    print("  ", end="")
    
    for a in range(i+1):
        print("#", end="")
    
    print("")


    



